pom.xml

```diff
@@ -1743,6 +1743,16 @@
                     <artifactId>provisio-maven-plugin</artifactId>
                     <version>1.0.18</version>
                 </plugin>
+
+                <plugin>
+                    <groupId>pl.project13.maven</groupId>
+                    <artifactId>git-commit-id-plugin</artifactId>
+                    <configuration>
+                        <runOnlyOnce>true</runOnlyOnce>
+                        <injectAllReactorProjects>true</injectAllReactorProjects>
+                        <offline>true</offline>
```

Comment:
What does offline mean?
Can you extract this change to separate PR to have it merged faster?
seems orthogonal to quick-related considerations

Reply:
@nineinchnick `offline` might not be needed since Airbase explicitly lists the properties it needs and none of them need a remote fetch. See https://github.com/airlift/airbase/pull/245/files

Reply:
https://github.com/trinodb/trino/pull/10521